hugo/content/sponsors.md

```diff
@@ -2,7 +2,24 @@
 title: Sponsors
 ---
 
-Our infrastructure is made possible thanks to the help of the sponsors that supported us over the years by providing their services for free.
+Our infrastructure is made possible thanks to the help of the following sponsors:
+
+## Cyberia
+
+<img src="/images/lain.png" alt="Let's all love Lain!" style="float:right; width: 380px; max-width: 40vw">
+
+The name should ring a bell if you have watched the masterpiece that is Serial Experiments Lain!
```

Comment:
Note: People unfamiliar with it will not know what that even is - that it is an anime.

Reply:
I mention it right below though:
> [Cyberia](https://cyberia.club) is managed by fans who decided to name their awesome collective after an important place in the anime.

Reply:
Maybe you could change it vice-versa. 
1. Explaning what Cyberia is, a hacker collective it seems.
2. Explaning that it refers to the anime.